flow/hooks/util.py

```diff
@@ -0,0 +1,37 @@
+# Copyright (c) 2023 The Regents of the University of Michigan
+# All rights reserved.
+# This software is licensed under the BSD 3-Clause License.
+"""Define a function to collect metadata on the operation and job."""
+from datetime import datetime, timezone
+
+
+def collect_metadata(operation, job):
+    """Collect metadata related to the operation and job.
+
+    Returns a directory including schema version, time, project, and job-operation.
+
+    We can no longer track the following
```

Comment:
The rest of this docstring looks incomplete and strangely wrapped (as if it's poetry?). Might need some reworking.